include/dxc/Support/WinIncludes.h

```diff
@@ -112,3 +114,26 @@ inline ENUMTYPE &operator ^= (ENUMTYPE &a, ENUMTYPE b) { return (ENUMTYPE &)(((_
 #endif
 
 #endif // _MSC_VER
+
+/// DxcCoGetMalloc
+#if defined(_WIN32) && !defined(DXC_DISABLE_ALLOCATOR_OVERRIDES)
+
+#define DxcCoGetMalloc CoGetMalloc
```

Comment:
Alternatively, we could always have a DxcCoGetMalloc function defined, which forwards to CoGetMalloc conditionally.  But this works too.

Reply:
I think this is fine, unless changing it would reduce dependencies or complexity somewhere.